client/wfprev-war/src/main/angular/package.json

```diff
@@ -23,10 +23,19 @@
     "@angular/platform-browser": "18.2.3",
     "@angular/platform-browser-dynamic": "18.2.3",
     "@angular/router": "18.2.3",
+    "@busacca/ng-pick-datetime": "^13.0.1",
+    "@ngrx/store": "^18.1.1",
     "@types/leaflet": "1.9.12",
+    "@wf1/core-ui": "^2.5.1",
+    "@wf1/incidents-rest-api": "^2.0.2",
```

Comment:
Why are we adding this? We don't use incident manager in Prev?

Reply:
Same as above. Removed